snark-keys/src/reader.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 ChainSafe Systems
```

Comment:
Same -- please hold off on this module until we are ready to start the verification milestone. This stuff is changing underneath.